server/src/configs/validators.rs

```diff
@@ -88,7 +88,8 @@ impl Validatable<ServerError> for MessageSaverConfig {
 impl Validatable<ServerError> for MessageCleanerConfig {
     fn validate(&self) -> Result<(), ServerError> {
         if self.interval == 0 {
-            panic!("Message cleaner interval must be greater than 0.")
+            // would be great if we can somehow add error message saying non-zero cleaner interval needs to be specified
```

Comment:
Maybe you could simply log an error message here e.g. similar to the segment size validation?

Reply:
done